package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "rails_api_base",
-  "private": "true",
+  "private": true,
   "engines": {
-    "node": "20.10.0",
-    "yarn": "1.22"
+    "node": "22.7.0",
```

Comment:
should we make this a little more flexible? something like what in Ruby would be `~22.0` ? So that as long as the major version matches we are good?